lib/ui/window/platform_configuration.h

```diff
@@ -582,8 +582,7 @@ class PlatformConfigurationNativeApi {
   static void RespondToPlatformMessage(int response_id,
                                        const tonic::DartByteData& data);
 
-  static void PlatformChannelListenedTo(const std::string& name,
-                                        bool listening);
+  static void SendChannelUpdate(const std::string& name, bool listening);
```

Comment:
Instead of using `const std::string&`, could we use `std::string` instead and `std::move` it down?

Reply:
Apparently, it seems this may not be an option. I tried that and pushed the change, but ironically, it seems doing so actually results in errors reported by the [clang tidy tests](https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket/8771483322002799041/+/u/test:_test:_lint_host_debug/stdout) explicitly saying to make the parameters const references. This is only the case for some of the lambdas/methods, but the signature of `PlatformView::SendChannelUpdate` must necessarily match that of `PlatformViewEmbedder::SendChannelUpdate`.